tests/helpers.ts

```diff
@@ -47,13 +49,57 @@ export const turboDevelopmentConfigurations = {
   },
 };
 
+// cspell:disable
 /**
  * Local wallet allow listed for tests
  */
-export const testWalletAddress = 'sYFSpEH7Gls-5Spq5FjuP85JCZj6QYzNvCm9BdKEJs4';
+export const testWalletAddress = 'sYFSpEH7Gls-5Spq5FjuP85JCZj6QYzNvCm9BdKEJs4'; // cspell:enable
 export const testJwk = JSON.parse(
   fs.readFileSync(
     new URL(`wallets/${testWalletAddress}.json`, import.meta.url).pathname,
     'utf-8',
   ),
 );
+
+const urlString = process.env.ARWEAVE_GATEWAY ?? 'http://localhost:1984';
+const arweaveUrl = new URL(urlString);
+export const testArweave = Arweave.init({
+  host: arweaveUrl.hostname,
+  port: +arweaveUrl.port,
+  protocol: arweaveUrl.protocol.replace(':', ''),
+});
+
+export async function fundArLocalWalletAddress(address: string): Promise<void> {
+  await testArweave.api.get(`mint/${address}/9999999999999999`);
+}
+
+export async function mineArLocalBlock(): Promise<void> {
+  await testArweave.api.get('mine');
```

Comment:
i didn't know this until recently - but `mine` API endpoint supports `/mine/{blocks}` if you need to mine multiple within one promise